tests/integration/targets/test_azure_virtual_machine_with_public_ip/tasks/main.yml

```diff
@@ -21,78 +25,168 @@
           subnet_address_prefixes_cidr:
           - "10.1.0.0/24"
 
-    # Test: Create Virtual machine with default options
-    - name: Create Virtual machine with default network interface
+    # Create VM with auto-created resorces within a new resource group.
+    # Stop, deallocate, start and restart VM
+    - name: Perform valid operations with VM within a new resource group
+      ansible.builtin.include_tasks: "azure_vm_tests.yml"
+      vars:
+        create_asserts_list:
```

Comment:
If I'm not wrong, most of the validations here are the same as in the second time
Why not moving all the validations to the imported file, except to the tags validation?

Reply:
Also the network interface name is different. It is already 2 additional checks. I thought to leave it near vars settings, in case in the future other parameters will be tested/modified.